packages/react/__tests__/__snapshots__/PublicAPI-test.js.snap

```diff
@@ -2525,9 +2525,6 @@ Map {
       "onClick": Object {
         "type": "func",
       },
-      "openCalendar": Object {
-        "type": "func",
-      },
```

Comment:
I don't think we can remove this without a major. We'll have to deprecate it instead.